docs/pages/api-reference/sources/kafka.md

```diff
@@ -30,7 +30,7 @@ servers (e.g. `localhost:9092,another.host:9092`).
 Protocol used to communicate with the brokers. 
 </Expandable>
 
-<Expandable title="sasl_mechanism" type="Optional[str]">
+<Expandable title="sasl_mechanism" type="str">
 SASL mechanism (e.g. SCRAM-SHA-256, PLAIN) to use for authentication.
```

Comment:
Can you please enumerate the valid SASL mechanism values that we support?

Reply:
Also are `sasl_plain_username` and `sasl_plain_password` still optional?

Reply:
Added list of SASL mechanisms supported. Also made sasl_plain_username and sasl_plain_password as non-optional